### PR TITLE
Octal To Binary Conerter

### PR DESCRIPTION
**Pull request by BamaCharanChhandogi**
- [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [X] This pull request is all my own work -- I have not plagiarized it.
- [X] All filenames are in PascalCase.
- [X] All functions and variable names follow Java naming conventions.
- [X] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

**octal To Binary Converter in Java.**

### CODE REVIEW

**Review by debasishbsws — CHANGES_REQUESTED** [review]
Add some JUnit tests, for reference see https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/conversions/OctalToDecimalTest.java

**Comment by BamaCharanChhandogi** [comment]
Ok, I have updated the code and added test cases.

File: src/test/java/com/thealgorithms/conversions/OctalToBinaryConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.thealgorithms.conversions;
```
Comment (siriak) [inline src/test/java/com/thealgorithms/conversions/OctalToBinaryConverterTest.java:1]: Please rename the file and the class to `OctalToBinaryTest`

File: src/main/java/com/thealgorithms/conversions/OctalToBinary.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter an octal number: ");
+        int octalNumber = scanner.nextInt();
+
+        // Convert octal to binary
+        long binaryNumber = convertOctalToBinary(octalNumber);
+
+        System.out.println("Binary equivalent: " + binaryNumber);
+    }
+
```
Comment (siriak) [inline src/main/java/com/thealgorithms/conversions/OctalToBinary.java:21]: Please remove `main`, it's redundant

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/conversions/OctalToBinary.java (hunk 2 above)
Comment (BamaCharanChhandogi) [reply src/main/java/com/thealgorithms/conversions/OctalToBinary.java:21]: All right, If I remove the main How to code will be run?
 I don't understand what is redundant

File: src/test/java/com/thealgorithms/conversions/OctalToBinaryConverterTest.java (hunk 1 above)
Comment (BamaCharanChhandogi) [reply src/test/java/com/thealgorithms/conversions/OctalToBinaryConverterTest.java:1]: Sorry for the mistake,

File: src/main/java/com/thealgorithms/conversions/OctalToBinary.java (hunk 2 above)
Comment (siriak) [reply src/main/java/com/thealgorithms/conversions/OctalToBinary.java:21]: This repository doesn't have an application that'd require an entry point. It's a library. If you want to call the provided functions, you can do so from your client code

Comment (BamaCharanChhandogi) [reply src/main/java/com/thealgorithms/conversions/OctalToBinary.java:21]: Please don't mind, I am just starting contributions to your repository. Can you please tell me what to do now? Just remove the main word.
thanks

File: src/main/java/com/thealgorithms/conversions/OctalToBinary.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
```
Comment (siriak) [inline src/main/java/com/thealgorithms/conversions/OctalToBinary.java:21]:
```suggestion
```

File: src/main/java/com/thealgorithms/conversions/OctalToBinary.java (hunk 2 above)
Comment (siriak) [reply src/main/java/com/thealgorithms/conversions/OctalToBinary.java]: I've deleted the `main()` function

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Review by debasishbsws — APPROVED** [review]
look good

**Pull request merged**